posix.mak

```diff
@@ -2,15 +2,26 @@ INSTALL_DIR=$(PWD)/../install
 ECTAGS_LANGS = Make,C,C++,D,Sh
 ECTAGS_FILES = src/*.[chd] src/backend/*.[chd] src/root/*.[chd] src/tk/*.[chd]
 
-.PHONY: all clean test install auto-tester-build auto-tester-test
+.PHONY: all clean test install auto-tester-build auto-tester-test toolchain-info
 
 all:
 	$(QUIET)$(MAKE) -C src -f posix.mak
 
-auto-tester-build:
+auto-tester-build: toolchain-info
 	$(QUIET)$(MAKE) -C src -f posix.mak auto-tester-build ENABLE_RELEASE=1
 
-auto-tester-test: test
+auto-tester-test: toolchain-info test
+
+toolchain-info:
+	@echo '==== Toolchain Information ===='
+	${MAKE} --version
+	${SHELL} --version || true
+	uname -a
+	c++ --version
+	ld -v
```

Comment:
Shouldn't these be ${CXX} and ${LD} rather than c++ and ld directly?  Also, how about adding a few lines like:
```
echo MAKE: ${MAKE}
```
to show their expansions?